libs/template/templates/lakeflow-pipelines/template/{{.project_name}}/databricks.yml.tmpl

```diff
@@ -36,11 +37,11 @@ targets:
     mode: production
     workspace:
       host: {{workspace_host}}
-      # We explicitly deploy to /Workspace/Users/{{user_name}} to make sure we only have a single copy.
-      root_path: /Workspace/Users/{{user_name}}/.bundle/${bundle.name}/${bundle.target}
-    permissions:
+      {{if not $isDLT}}# We explicitly deploy to /Workspace/Users/{{user_name}} to make sure we only have a single copy.
+      root_path: /Workspace/Users/{{user_name}}/.bundle/${bundle.name}/${bundle.target}}{{end}}
+    {{if $isDLT}}owner: user@company.com{{else}}permissions:
```

Comment:
Please keep in mind that the template won't yet work if you replace the root_path with the owner as I originally suggested. The CLI would first need to support that.

So you should probably sequence this change to be later.